src/platform/datapath_winkernel.c

```diff
@@ -471,7 +471,7 @@ CxPlatDataPathQuerySockoptSupport(
         Datapath->WskProviderNpi.Dispatch->
         WskSocket(
             Datapath->WskProviderNpi.Client,
-            AF_INET,
+            AF_INET6,
```

Comment:
Why did you make this change?

Reply:
IPv4 silently fails on previous versions of Windows, while IPv6 fails immediately. 
I could open a separate IPv6 socket just for the DSCP feature test, but it's a lot of extra boilerplate code